x11-misc/compiz-debug-utils/compiz-debug-utils-0_pre20171023.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+myCommit="f700dd2b6af1f0c93b6b1eb1d75e658783f848e2"
+
+DESCRIPTION=" Helper utilities for debugging Compiz"
+HOMEPAGE="https://github.com/compiz-reloaded"
+SRC_URI="https://github.com/compiz-reloaded/${PN}/archive/${myCommit}.zip -> ${P}-${myCommit}.zip"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+		>=dev-lang/python-3:*
```

Comment:
Using Python is allowed only via python*r1 eclasses. I think python-single-r1 would be a good fit here. Also, too much indent.